src/script/lua_api/l_async.cpp

```diff
@@ -0,0 +1,112 @@
+#include "lua_api/l_internal.h"
```

Comment:
missing license header

Reply:
Done. I was not exactly confident in adding the header (esp. with self-attribution) since the code is largely copied from elsewhere in the codebase.

Reply:
You can also use the short form we use in some files:
```c
// Minetest
// SPDX-License-Identifier: LGPL-2.1-or-later
```